week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -42,20 +42,76 @@ My sources say no.
 Outlook not so good.
 Very doubtful.
 */
-
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+
```

Comment:
You made mistakes in this part of the exercise. you should have a possible answer which holds the different possible answers.